.prettierrc

```diff
@@ -1,6 +1,6 @@
 {
```

Comment:
I want to say we used to have this as just an `*rc` file, but moved to the `*.json` naming due to an issue with `prettier`/`pretty-quick` not picking it up. Did they fix that and is it being picked up?

Reply:
From my testing at least it was working this way. not sure if it's not working on your end